### ISSUE TITLE
GymEnv with Normalization

### ISSUE THREAD

**Issue by atavakol** [openai/rllab#160]
For running Gym tasks with GymEnv, do I need to use normalize()?

**Comment by dementrock** [comment]
The action spaces should already be normalized, so no. However normalizing the observations may still improve performance.

**Comment by atavakol** [comment]
Thanks for your response.

**Comment by aravindsrinivas** [comment]
@dementrock  How about the reward normalization?  Doesn't it affect the evaluation (since the mean_reward and sdev_reward are still being updated over training, and the metrics are kind of non-stationary)?

**Comment by dementrock** [comment]
Yes. Reward normalization should mostly be avoided (it was mostly an experimental feature).
